pages/_en-US/twilightmenu/controls.md

```diff
@@ -41,9 +41,15 @@ These do not apply to DSiWare.
 - <kbd class="l">L</kbd> + <kbd class="r">R</kbd> + <kbd>Down</kbd> + <kbd class="face">A</kbd> for 3 seconds: Dump RAM to `sd:/_nds/nds-bootstrap`, as `ramDump.bin`
 - <kbd class="l">L</kbd> + <kbd class="r">R</kbd> + <kbd>Down</kbd> + <kbd class="face">B</kbd> for 2 seconds: Return to TWiLight Menu++
 - <kbd class="l">L</kbd> + <kbd>Down</kbd> + <kbd>SELECT</kbd>: Open the in-game menu
+   - <kbd class="l">L</kbd>: Advance by one frame
+   - Screenshot
+      - <kbd>Left</kbd>/<kbd>Right</kbd>: Change VRAM bank
```

Comment:
<kbd>Up</kbd> and <kbd>Down</kbd> do the same thing as <kbd>Left</kbd> and <kbd>Right</kbd> respectively here, otherwise looks good